src/code/z_camera_data.c

```diff
@@ -35,10 +35,36 @@ s16 sOREGInit[] = {
 s16 sOREGInitCnt = 53;
 
 s16 sPREGInit[] = {
-    -20, 200, 300, 10, 12, 10, 35, 60, 60, 3, 0, -40, 20, 25, 45, -5, 15, 15, 20, 0, 0, 0, 6, 60, 30, 0, 5,
-};
-
-s16 sPREGInitCnt = 27;
+    -20, // R_CAM_DATA(CAM_DATA_Y_OFFSET)
+    200, // R_CAM_DATA(CAM_DATA_EYE_DIST)
+    300, // R_CAM_DATA(CAM_DATA_EYE_DIST_NEXT)
+    10,  // R_CAM_DATA(CAM_DATA_PITCH_TARGET)
+    12,  // R_CAM_DATA(CAM_DATA_YAW_UPDATE_RATE_TARGET)
+    10,  // R_CAM_DATA(CAM_DATA_XZ_UPDATE_RATE_TARGET)
+    35,  // R_CAM_DATA(CAM_DATA_MAX_YAW_UPDATE)
+    60,  // R_CAM_DATA(CAM_DATA_FOV)
+    60,  // R_CAM_DATA(CAM_DATA_AT_LERP_STEP_SCALE)
+    3,   // R_CAM_DATA(CAM_DATA_FLAGS)
+    0,   // R_CAM_DATA(CAM_DATA_YAW_TARGET)
+    -40, // R_CAM_DATA(CAM_DATA_GROUND_Y_OFFSET)
+    20,  // R_CAM_DATA(CAM_DATA_GROUND_AT_LERP_STEP_SCALE)
+    25,  // R_CAM_DATA(CAM_DATA_SWING_YAW_INIT)
+    45,  // R_CAM_DATA(CAM_DATA_SWING_YAW_FINAL)
+    -5,  // R_CAM_DATA(CAM_DATA_SWING_PITCH_INIT)
+    15,  // R_CAM_DATA(CAM_DATA_SWING_PITCH_FINAL)
+    15,  // R_CAM_DATA(CAM_DATA_SWING_PITCH_ADJ)
+    20,  // R_CAM_DATA(CAM_DATA_MIN_MAX_DIST_FACTOR)
+    0,   // R_CAM_DATA(CAM_DATA_AT_OFFSET_X)
+    0,   // R_CAM_DATA(CAM_DATA_AT_OFFSET_Y)
+    0,   // R_CAM_DATA(CAM_DATA_AT_OFFSET_Z)
+    6,   // R_CAM_DATA(CAM_DATA_UNK_22)
+    60,  // R_CAM_DATA(CAM_DATA_UNK_23)
+    30,  // R_CAM_DATA(CAM_DATA_FOV_SCALE)
+    0,   // R_CAM_DATA(CAM_DATA_YAW_SCALE)
+    5,   // R_CAM_DATA(CAM_DATA_UNK_26)
+};
+
+s16 sPREGInitCnt = CAM_DATA_MAX;
```

Comment:
```suggestion
s16 sPREGInitCnt = ARRAY_COUNT(sPREGInit);
```
this would go together with my next comment